include/RAJA/policy/cuda/kernel/internal.hpp

```diff
@@ -101,7 +101,7 @@ namespace internal
 {
 
 RAJA_INLINE
-int get_size(cuda_dim_t dims)
+size_t get_size(cuda_dim_t dims)
```

Comment:
The items multiplied still need to be size_t before multiplication to avoid overflow